docs/translations/api-docs/rating/rating.json

```diff
@@ -12,7 +12,7 @@
     "IconContainerComponent": "The component containing the icon.",
     "max": "Maximum rating.",
     "name": "The name attribute of the radio <code>input</code> elements. This input <code>name</code> should be unique within the page. Being unique within a form is insufficient since the <code>name</code> is used to generated IDs.",
-    "onChange": "Callback fired when the value changes.<br><br><strong>Signature:</strong><br><code>function(event: object, value: number) =&gt; void</code><br><em>event:</em> The event source of the callback.<br><em>value:</em> The new value.",
+    "onChange": "Callback fired when the value changes.<br><br><strong>Signature:</strong><br><code>function(event: object, value: number \\| null) =&gt; void</code><br><em>event:</em> The event source of the callback.<br><em>value:</em> The new value.",
```

Comment:
Any idea why he `|` is escaped? It produces a rogue `\` in the deployed docs: https://deploy-preview-26542--material-ui.netlify.app/api/rating/

Reply:
Continued in https://github.com/mui-org/material-ui/issues/26547